ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -1497,6 +1522,29 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ipi-gcp
+- as: amd-notebooks-e2e-tests
```

Comment:
Right now, we have test only for tensorflow, you will need to do the same for pytorch .
Hint: copy the lines from 1530-1546 and replace it accordingly